### PR TITLE
Formatted code; Added goimports to Makefile, Renamed pb_testproto to testpb.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
Should be good for review @johanbrandhorst (: Thanks!

File: grpctesting/pingservice.go [hunk 1]
```diff
@@ -12,7 +12,8 @@ import (
 	"io"
 	"testing"
 
-	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testproto"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testpb"
+
```
Comment (johanbrandhorst) [inline grpctesting/pingservice.go:16]: I think we have to manually move this to the bottom

File: interceptors/logging/interceptors_test.go [hunk 2]
```diff
@@ -9,22 +9,22 @@ import (
...
-	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/tags"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 
-	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testproto"
+	middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting"
```
Comment (johanbrandhorst) [inline interceptors/logging/interceptors_test.go:19]: These should be together

File: interceptors/server_test.go [hunk 3]
```diff
@@ -12,13 +12,13 @@ import (
...
 	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting"
-	"google.golang.org/grpc/status"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testpb"
 
-	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testproto"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
```
Comment (johanbrandhorst) [inline interceptors/server_test.go:21]: Reorder these

File: interceptors/tags/fieldextractor_test.go [hunk 4]
```diff
@@ -7,29 +7,30 @@ import (
...
 
-	pb_gogotestproto "github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/gogotestproto"
-	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testproto"
-	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/tags"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/gogotestpb"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testpb"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/tags"
```
Comment (johanbrandhorst) [inline interceptors/tags/fieldextractor_test.go:15]: Reorder these

File: interceptors/tracing/interceptors_test.go [hunk 5]
```diff
@@ -22,25 +21,27 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 
+	middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testpb"
+
 	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting"
-	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testproto"
 	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/tags"
 	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/tracing"
```
Comment (johanbrandhorst) [inline interceptors/tracing/interceptors_test.go:29]: Merge these

**Comment by bwplotka** [comment]
Easy to mess this so true

**Comment by bwplotka** [comment]
Addresed all (:

**Pull request merged**